bulletin_board/server/spec/commands/create_election_spec.rb

```diff
@@ -87,7 +87,27 @@
     it_behaves_like "create election fails"
 
     it "broadcasts invalid" do
-      expect { subject }.to broadcast(:invalid, "Election should start at least in 2 hours from now.")
+      expect { subject }.to broadcast(:invalid, "Election should start at least in 1 hours from now.")
+    end
+
+    context "when HOURS_BEFORE_CREATE_ELECTION is set" do
```

Comment:
As we're not doing the test with the ENV VAR, I'd prefer to not reference to that in the example title

```suggestion
    context "when the `hours before` the election creation is set" do
```